02.general/17.dns-providers/docs.en.md

```diff
@@ -0,0 +1,425 @@
+---
+title: 'Known DNS Providers'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+
+AdGuard users can configure any DNS server to be used instead of the system default provided by the router or ISP. In this article you will find a list of popular DNS providers.
+
+* [AdGuard DNS](#adguard-dns)
+* [Yandex DNS](#yandex-dns)
+* [CleanBrowsing](#cleanbrowsing)
+* [Neustar Recursive DNS](#neustar-recursive-dns)
+* [Cisco OpenDNS](#cisco-opendns)
+* [Google DNS](#google-dns)
+* [Cloudflare DNS](#cloudflare-dns)
+* [Quad9 DNS](#quad9)
+* [Verisign Public DNS](#verisign-public-dns)
+* [DNS.WATCH](#dnswatch-dns)
+* [Comodo Secure DNS](#comodo-secure-dns)
+* [Dyn DNS](#dyn-dns)
+* [OpenNIC DNS](#opennic-dns)
+* [Quad101](#quad101)
+* [Freenom World](#freenom-world)
+* [Fortinet Secure](#fortinet-secure)
+* [Safe DNS](#safe-dns)
+* [Strongarm DNS](#strongarm-dns)
+* [SafeSurfer DNS](#safesurfer-dns)
+* [Captnemo DNS](#captnemo-dns)
+* [fvz DNS](#fvz-dns)
+* [Nawala Childprotection DNS](#nawala-childprotection-dns)
+
+<a name="adguard-dns"></a>
+
+### AdGuard DNS
+
+[AdGuard DNS](https://adguard.com/adguard-dns/overview.html) is an alternative solution for ad blocking, privacy protection, and parental control. It provides the number of necessary protection features against online ads, trackers, and phishing, no matter what platform and device you use.
+
+#### Default
+
+These servers provide blocking ads, tracking and phishing
+
+| Protocol       | Address                                            |                |
+|----------------|----------------------------------------------------|----------------|
+| DNS, IPv4      | `176.103.130.130` and `176.103.130.131`             | <a href="sdns://regular&176.103.130.130+176.103.130.131">Add to AdGuard</a> |
+| DNS, IPv6      | `2a00:5a60::ad1:0ff` and `2a00:5a60::ad2:0ff`            | <a href="sdns://regular&2a00:5a60::ad1:0ff+2a00:5a60::ad2:0ff">Add to AdGuard</a> |
+| DNSCrypt, IPv4 | Provider: `2.dnscrypt.default.ns1.adguard.com` IP: `176.103.130.130:5443`| <a href="sdns://AQIAAAAAAAAAFDE3Ni4xMDMuMTMwLjEzMDo1NDQzINErR_JS3PLCu_iZEIbq95zkSV2LFsigxDIuUso_OQhzIjIuZG5zY3J5cHQuZGVmYXVsdC5uczEuYWRndWFyZC5jb20">Add to AdGuard</a> |
+| DNSCrypt, IPv6 |  Provider: `2.dnscrypt.default.ns2.adguard.com` IP: `[2a00:5a60::ad2:0ff]:5443`| <a href="sdns://AQIAAAAAAAAAGVsyYTAwOjVhNjA6OmFkMjowZmZdOjU0NDMggdAC02pMpQxHO3R5ZQ_hLgKzIcthOFYqII5APf3FXpQiMi5kbnNjcnlwdC5kZWZhdWx0Lm5zMi5hZGd1YXJkLmNvbQ">Add to AdGuard</a> |
+| DNS-over-HTTPS | Provider: `https://dns.adguard.com/dns-query`, IP: `176.103.130.130` | <a href="sdns://AgcAAAAAAAAADzE3Ni4xMDMuMTMwLjEzMAAPZG5zLmFkZ3VhcmQuY29tCi9kbnMtcXVlcnk">Add to AdGuard</a> |
```

Comment:
For DoH don't just supply the URL: `https://dns.adguard.com/dns-query`

IP / Provider aren't necessary.